addons/im_livechat/static/src/embed/common/boot_helpers.js

```diff
@@ -47,6 +59,14 @@ export function makeRoot(target) {
     return root;
 }
 
+export async function loadAssets(styleTarget, fontsTarget = document) {
+    await Promise.all([
+        loadStyle(styleTarget),
+        loadFont("FontAwesome", url("/im_livechat/font-awesome"), fontsTarget),
+        loadFont("odoo_ui_icons", url("/im_livechat/odoo_ui_icons"), fontsTarget),
+    ]);
+}
```

Comment:
`fontsTarget` is unnecessary parameter: can always use the `document` as target.